src/app/profile/settings/resources/resources.component.less

```diff
@@ -0,0 +1,28 @@
+@import (reference) '../../../../assets/stylesheets/shared/osio.less';
+
+.resources-center-text {
+  text-align: center;
+}
+
+.resources-remove-margin {
+  margin-bottom: 0;
+}
+
+.resources-full-capacity {
+  background-color: #f9e5e5;
```

Comment:
Use PatternFly color variable.

Reply:
It doesn't seem like there is one ? The colour is trying to match the background on the entry when full capacity is reached, as seen here https://redhat.invisionapp.com/share/FWFGF7VXMB8#/screens/282415828